int16 pos_x
int16 pos_y
int16 theta
int8 ball_status
int16 ball_x
int16 ball_y
uint16 robot_cond
int8 pass_target